build.Dockerfile

```
@@ -4,26 +4,21 @@ FROM ubuntu:focal AS build
 ### Build stage
 
 # Install curl and git and smp-related dependencies
-RUN apt-get update && apt-get install -y curl git build-essential libgmp3-dev zlib1g-dev
+RUN apt-get update && apt-get install -y curl git build-essential libgmp3-dev zlib1g-dev llvm-11 llvm-11-dev libnuma-dev
```

Let's not specify exact `LLVM` version and use whatever Ubuntu provides by default (focal (20.04LTS) use `llvm-10`).

```suggestion
RUN apt-get update && apt-get install -y curl git build-essential libgmp3-dev zlib1g-dev llvm llvm-dev libnuma-dev
```